charts/rasa-x/values.yaml

```
@@ -67,6 +67,8 @@ rasa:
   initialProbeDelay: 10
   # lockStoreDatabase is the database in redis which Rasa uses to store the conversation locks
   lockStoreDatabase: "1"
+  # extra env variables
```

let's follow the Helm best practices and start the comment with the same name as the variable:
```suggestion
  # extraEnvs are environment variables which can be added to the Rasa X deployment
```

---

Agree with the suggestion of following best practices here